src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -0,0 +1,107 @@
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+import static com.google.common.collect.FluentIterable.from;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class Exercise1 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortComparator() {
+        Person[] persons = getPersons();
+        Arrays.sort(persons, Comparator.comparingInt(Person::getAge));
+        assertArrayEquals(new Person[]{
+                new Person("Иван", "Мельников", 20),
+                new Person("Николай", "Зимов", 30),
+                new Person("Алексей", "Доренко", 40),
+                new Person("Артем", "Зимов", 45)
+        }, persons);
+    }
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortAnonymousComparator() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, new Comparator<Person>() {
+            @Override
+            public int compare(Person o1, Person o2) {
+                return Integer.compare(o1.getAge(), o2.getAge());
+            }
+        });
+        assertArrayEquals(new Person[]{
+                new Person("Иван", "Мельников", 20),
+                new Person("Николай", "Зимов", 30),
+                new Person("Алексей", "Доренко", 40),
+                new Person("Артем", "Зимов", 45)
+        }, persons);
+    }
+
+    @Test
+    public void sortPersonsByLastNameThenFirstNameUsingArraysSortAnonymousComparator() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, new Comparator<Person>() {
+            @Override
+            public int compare(Person o1, Person o2) {
+                return ComparisonChain.start()
+                        .compare(o1.getLastName(), o2.getLastName())
+                        .compare(o1.getFirstName(), o2.getFirstName())
+                        .result();
+            }
+        });
+        assertArrayEquals(new Person[]{
+                new Person("Алексей", "Доренко", 40),
+                new Person("Артем", "Зимов", 45),
+                new Person("Николай", "Зимов", 30),
+                new Person("Иван", "Мельников", 20)
+        }, persons);
+    }
+
+    @Test
+    public void findFirstWithAge30UsingGuavaPredicate() {
+        List<Person> persons = Arrays.asList(getPersons());
+
+        Person person = from(persons).firstMatch(pers -> pers.getAge() == 30).get();
+
+        assertEquals(new Person("Николай", "Зимов", 30), person);
+    }
+
+    @Test
+    public void findFirstWithAge30UsingGuavaAnonymousPredicate() {
+
+        List<Person> persons = Arrays.asList(getPersons());
+
+        Predicate<Person> predicate = new Predicate<Person>() {
+            @Override
+            public boolean apply(Person person) {
+                if (person.getAge() == 30) {
+                    return true;
+                } else {
+                    return false;
+                }
```

Comment:
It's better to use ternary operator in cases like this. 
At this case we don't need even ternary operator: `return person.getAge() == 30;`